cmd/simulacra/README.md

```diff
@@ -0,0 +1,70 @@
+# Simulacra 
+
+Simulacra is a command line tool that the oncall team can use to automatically spin up VMs with a customers environment. 
+
+Simulacra can read Ops Agent config files and infer all the third party apps that it needs to install. It then instantiates the VM according to user specifications, installs Ops Agent and then installs all the required third party applications. 
```

Comment:
This paragraph being one of the first things said is a bit confusing. I'd start with something more basic like "Simulacra can be configured through command line for quick operations, or a config file for a reproducible setup" and then go into how it can also read a config file to infer apps.